docker-compose.yml

```diff
@@ -9,8 +9,28 @@ x-services:
       - postgres
       - minio
     environment:
-      DATABASE_URI: postgresql+psycopg2://$DB_USER:$DB_PASSWORD@$DB_HOST:$CONN_DB_PORT/$DB_NAME
-      API_PORT: $API_PORT
+      - DATABASE_URI=postgresql+psycopg2://$DB_USER:$DB_PASSWORD@$DB_HOST:$CONN_DB_PORT/$DB_NAME
```

Comment:
at this point it may be eaiser just to use
```
env_file:
   - .env
```

to make things shorter (like intergov service does)

Reply:
I'm tidying, consistencising, de-duplicating furiously. I did have problems with env_file not working as expected though. Was it that env vars in the env_file aren't available in the .yml file? Anyway, it also sucks that it errors if the file is missing; ie. you can't have optional .env files.

In short, I share your frustrations and am working to fix them.